CHANGELOG.adoc

```diff
@@ -66,6 +66,9 @@ This allows downstream projects and applications to provide their own sub-types
 In addition to registering the extension metadmodel itself, users must provide a `ChildExtenderProvider` bean for their extensions to be properly integrated.
 - https://github.com/eclipse-sirius/sirius-components/issues/1918[#1918] [tree] Its is now possible to filter tree items in trees.
 After selected a tree item, hit Ctrl+f (or Cmd+f on macOS) to enable the filter bar.
+- https://github.com/eclipse-sirius/sirius-components/issues/2015[#2015] [form] We add to the payload the possibility to return a list of messages, these messages will be displayed
```

Comment:
- 1 sentence == 1 physical line
- the new text is inserted between the "Filter bar" text (above) and the corresponding screenshot (below).